### ISSUE TITLE
Access channel pages of creators in subscription groups

### ISSUE THREAD

**Issue by PhilC813** [futo-org/grayjay-android#1563]
### Use Cases

I have subscribed to a few key channels that I am using not for following every new upload, but rather simply to search through (contextual channel search) since they are large collections of practical content.

Some examples:
| Channel | Content type |
|--------|--------|
| [IGN Guides](https://youtube.com/@IGNGuides) | Game walkthroughs |
| [Rotten Tomatoes Trailers](https://youtube.com/@RottenTomatoesTRAILERS) | Trailers for recent movies |
| [Rotten Tomatoes Classic Trailers](https://youtube.com/@RottenTomatoesCLASSICTRAILERS) | Trailers for older movies |

I then organized them in subscription groups:
| Group | Channel(s) |
|--------|--------|
| Video games | • [IGN Guides](https://youtube.com/@IGNGuides) |
| Movies | • [Rotten Tomatoes Trailers](https://youtube.com/@RottenTomatoesTRAILERS)<br>• [Rotten Tomatoes Classic Trailers](https://youtube.com/@RottenTomatoesCLASSICTRAILERS) |

I was expecting to be able to use subscription groups to easy access the individual channels within them, and then perform the channel-specific video search. However, with the current implementation, subscription groups only allow the following:
- Tap on the group | Filter the subscriptions tab to only show videos from creators that are part of the group
- Long tap on the group | Accesses the settings of the group, which lets you add or remove channels, but not access one of their pages.

In other words, subscription groups provide no way to simply access the page of one of the member channels.

### Proposal

Two potential ways to solve it:
- Add the capability to access the page of individual channels within subscription groups
- Create a different type of group within the "Creators" tab, which would be specifically designed to organize creator profiles.
